site/versioning-policy.html

```diff
@@ -187,9 +187,9 @@ <h3>Alpha components</h3>
 
 <h3>API compatibility</h3>
 
-<p>An API is any public class or interface exposed in Spark that is not marked as &#8220;developer API&#8221; or 
-&#8220;experimental&#8221;. Release A is API compatible with release B if code compiled against release A 
-<em>compiles cleanly</em> against B. Currently, does not guarantee that a compiled application that is 
+<p>In general, an API is any public class or interface documented in Spark.
+Release A is API compatible with release B if code compiled against release
+A <em>compiles cleanly</em> against B. Currently, does not guarantee that a compiled application that is
```

Comment:
not related to this PR, but this statement seems to indicate that we don't guarantee binary compatibility?

Reply:
Actually yeah. Let me reword a little bit.